### ISSUE TITLE
Unicode subscript render bug in cell output

### ISSUE THREAD

**Issue by chrisoffner3d** [fonsp/Pluto.jl#1754]
Hey everyone! Not entirely sure whether this is an issue that can be fixed on Pluto's end or it's some Unicode weirdness outside of Pluto's control, but when I try to create a docstring containing multiple subscript characters in a row, Pluto renders it differently from the way it looks in the cell:

## Video
https://user-images.githubusercontent.com/18167754/145387140-01711c04-e05b-4709-b054-5c9f32ff7ff8.mp4

## Example notebook

https://gist.github.com/chrisoffner/55cca40d8d75da977de4e5fa74295511

**Comment by fonsp** [comment]
I agree that it looks bad but somehow this is correct, I checked it against other systems.
